src/visitpy/CMakeLists.txt

```diff
@@ -125,11 +119,10 @@ SET(PY_GEN_STATE_OBJECTS
     AxisAttributes
     AxisLabels
     AxisTickMarks
-    AxisTitle
```

Comment:
Yes, `AxisTitles` (note the `s` at the end) is a real thing here and was/is near bottom of list. Maybe I should move towards top?

However, `AxisTitle` (no ending `s`) does not exist AFAICT. I did multiple attempts to find it and it doesn't exist. Seemed like a typo that was benign but nonetheless misleadig. Do you have any info to suggest its a real thing?

Reply:
BTW...this is just one very simple example that demonstrates one cannot always understand a change by looking at *just* the changed lines. I often find myself expanding more of the file (in diff view on GitHub) or going to `...->View File` before I understand a change well enough to make an informed decision. I mention this because I have fallen into this trap too and I think we, @visit-dav/visit-developers, should all take care to put a slightly different thinking cap on when reviewing changes...sometimes you need to look at a lot more stuff *around* a change but which itself isn't part of the change to fully understand it. Not trying to be critical. Just hoping to be helpful by pointing out a common pitfall 😉